instance/hasgeekapp-sample.py

```diff
@@ -12,3 +11,8 @@
 ASSET_MANIFEST_PATH = "static/build/manifest.json"
 
 SESSION_COOKIE_NAME = 'root_session'
+DELETE_COOKIES = {
+    'session': {'domain': '.hasgeekapp.local', 'without_dot': True},
+    'root_session': {'domain': '.hasgeekapp.local', 'without_dot': False},
```

Comment:
Don't use `without_dot`, just list domains. We may have a future need for specific paths as well.